examples/client.php

```diff
@@ -0,0 +1,434 @@
+
+        $ngnBankDetails = new PayoutMethodDetails();
+        $ngnBankDetails->setBankAccount('123456789');
+        $ngnBankDetails->setBankAccountType(PayoutMethodBankAccountTypeEnum::_20);
+        $ngnBankDetails->setBankCode('082');
+        $ngnBankDetails->setFirstName('First');
+        $ngnBankDetails->setLastName('Last');
+
+        $payoutMethod = new PayoutMethod();
+        $payoutMethod->setType('NGN::Bank');
+        $payoutMethod->setDetails($ngnBankDetails);
+
+        $recipient = new Recipient();
+        $recipient->setRequestedAmount(10000);
+        $recipient->setRequestedCurrency('NGN');
+        $recipient->setPayoutMethod($payoutMethod);
+
+        $transaction->setInputCurrency('GBP');
+        $transaction->setSender($sender);
+        $transaction->setRecipients([$recipient]);
+
+        try {
+            $transactionRequest = new TransactionRequest();
+            $transactionRequest->setTransaction($transaction);
+            $transactionResponse = $api->postTransactions($transactionRequest);
+            $createdTransaction = $transactionResponse->getObject();
+            echo "Transaction created! ID: {$createdTransaction->getId()}", PHP_EOL;
+            return $createdTransaction->getId();
+        } catch (ApiException $e) {
+            if ($e->isValidationError()) {
+                $response = $e->getResponseObject();
+                $validation_errors = $response->getObject()->getErrors();
+                print_r($validation_errors);
+            } else {
+                throw $e;
+            }
+            return null;
+        }
+    }
+
+    public function createAndFundTransactionExample() {
+        $transactionId = $this->createTransactionExample();
+
+        if (!empty($transactionId)) {
+            $debit = new Debit();
+            $debit->setCurrency('GBP');
+            $debit->setToId($transactionId);
+            $debit->setToType("Transaction");
+
+            $debitRequest = new DebitRequestWrapper();
+            $debitRequest->setDebit($debit);
+
+            $debitApi = new AccountDebitsApi();
+            try {
+                $debitListResponse = $debitApi->postAccountsDebits($debitRequest);
+                echo "Transaction Funded Successfully" . PHP_EOL;
+                print_r($debitListResponse->getObject()[0]);
+            } catch (ApiException $e) {
+                if ($e->isValidationError()) {
+                    $response = $e->getResponseObject();
+                    echo "Transaction could not be funded" . PHP_EOL;
+                    $validation_errors = $response->getObject()[0]->getErrors();
+                    print_r($validation_errors);
+                } else {
+                    throw $e;
+                }
+            }
+        }
+        return $transactionId;
+    }
+
+    public function createSenderExample() {
+        $docData = file_get_contents('./bitpesa-data-uri.txt');
+        $document = new Document();
```

Comment:
No need to support document upload, in fact please change it to an example where we only upload an empty array, which will be used 99% if the cases